packages/web-js-sdk/src/enhancers/withPersistTokens/helpers.ts

```diff
@@ -34,40 +34,46 @@ function setJwtTokenCookie(
 
 export const persistTokens = (
   { refreshJwt, sessionJwt, ...cookieParams } = {} as Partial<JWTResponse>,
-  sessionTokenViaCookie = false
+  sessionTokenViaCookie = false,
+  storagePrefix = ''
 ) => {
   // persist refresh token
-  refreshJwt && setLocalStorage(REFRESH_TOKEN_KEY, refreshJwt);
+  refreshJwt &&
+    setLocalStorage(`${storagePrefix}${REFRESH_TOKEN_KEY}`, refreshJwt);
 
   // persist session token
   if (sessionJwt) {
     sessionTokenViaCookie
       ? setJwtTokenCookie(SESSION_TOKEN_KEY, sessionJwt, cookieParams)
-      : setLocalStorage(SESSION_TOKEN_KEY, sessionJwt);
+      : setLocalStorage(`${storagePrefix}${SESSION_TOKEN_KEY}`, sessionJwt);
```

Comment:
we should also handle the cookie value here in the same manner, this solution should not be narrowed only for local storage

in general I think that introducing such different behavior for cookie and local storage is something we should avoid

Reply:
if we change the cookies nothing will work. mind this is a storage prefix, while cookies should be managed somewhere else including cookies per tenant etc, we cant change the cookie name